libs/libcommon/src/libcommon/queue.py

```diff
@@ -854,15 +847,15 @@ def get_pending_jobs_df(self, dataset: str, job_types: Optional[List[str]] = Non
         return self._get_df(
             [
                 job.flat_info()
-                for job in JobDocument.objects(dataset=dataset, status__in=[Status.WAITING, Status.STARTED], **filters)
+                for job in JobDocument.objects(status__in=[Status.WAITING, Status.STARTED], **filters, dataset=dataset)
```

Comment:
does the order matter here ?

Reply:
I don't think it will improve performance, but I changed it because I saw that we are using the same index but in a different order in other parts of the code. But, would like to try (In other DB engines, the column order of the filter/select affects the best query plan choice).